app/src/main/AndroidManifest.xml

```diff
@@ -7,17 +7,13 @@
         android:glEsVersion="0x00020000"
         android:required="true"/>
 
-    <uses-feature
```

Comment:
I see the following error in my IDE on this line.  I removed this entry since the effect of marking the attribute `required` as false is equivalent to not stating the hardware use. Let me know what you think.

`Expecting android:required="false" for this hardware feature that may not be supported by all Chrome OS devices`.